Classes/YTPlayerView.h

```diff
@@ -602,9 +616,10 @@ typedef NS_ENUM(NSInteger, YTPlayerError) {
  * JavaScript API defined here:
  *   https://developers.google.com/youtube/iframe_api_reference#getPlaybackQuality
  *
- * @return YTPlaybackQuality representing the current playback quality.
+ * @param completionHandler async callback block that contains a YTPlaybackQuality enum value
+ * with the current player state or an error.
  */
-- (YTPlaybackQuality)playbackQuality;
+- (void)playbackQuality:(_Nullable YTPlaybackQualityCompletionHandler)completionHandler;
```

Comment:
All of the playback quality functions have been removed from the iFrame API, see "revision history" from 2019-10-24:
https://developers.google.com/youtube/iframe_api_reference#october-24,-2019

We should take this opportunity to remove from the player-helper lib.

Reply:
done